### PR TITLE
Change language names to the names on Google Translate

### PR DESCRIPTION
**Pull request by y2-Whyto**
## Description

Changed some language names to the names on Google Translate.

This extension is used for translation, and should not contain political attributes. So removed contents potentially causing disputations.

## Screencast

None

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by gebeto** [comment]
This PR is related to https://github.com/raycast/extensions/pull/1145

@neighborhood999 what do you think about current change?

**Comment by neighborhood999** [comment]
> This PR is related to #1145
> 
> @neighborhood999 what do you think about current change?

From my perspective, I don't think my PR has *political* attributes.

**Comment by y2-Whyto** [comment]
> > This PR is related to #1145
> 
> > 
> 
> > @neighborhood999 what do you think about current change?
> 
> 
> 
> From my perspective, I don't think my PR has *political* attributes.


@gebeto ,

However, the name of it on Google Translate itself is "Chinese (Traditional)". Moreover, this name is obviously more widely acknowledged than the language name in this plugin at present.

Also, this is not only a matter of politics, but also (and even more) of **acceptance**. Though maybe you made this modification out of no political reason, that really doesn't ensure that others won't have prejudice upon that description. After all, there's strong regional meaning in the current name. We are just talking about **languages** here. Plugin users who purely use traditional Chinese and do not live there will possibly feel uncomfortable and excluded about that name.

So I hope you could consider more about the issues I mention above. Thank you.

**Review by gebeto — APPROVED** [review]

File: extensions/google-translate/src/supportedLanguages.json [hunk 1]
```diff
@@ -70,8 +70,7 @@
   },
   "Mandarin": {
     "code": "zh-TW",
-    "name": "Taiwanese Mandarin",
-    "flag": "🇹🇼"
+    "name": "Chinese Traditional",
```
Comment (gebeto) [inline extensions/google-translate/src/supportedLanguages.json:73]:
```suggestion
    "name": "Chinese Traditional"
```

File: extensions/google-translate/src/supportedLanguagesByCode.json [hunk 2]
```diff
@@ -70,8 +70,7 @@
   },
   "zh-TW": {
     "code": "zh-TW",
-    "name": "Taiwanese Mandarin",
-    "flag": "🇹🇼"
+    "name": "Chinese Traditional",
```
Comment (gebeto) [inline extensions/google-translate/src/supportedLanguagesByCode.json:73]:
```suggestion
    "name": "Chinese Traditional"
```

File: extensions/google-translate/src/supportedLanguages.json (hunk 1 above)
Comment (gebeto) [reply extensions/google-translate/src/supportedLanguages.json:73]: Please fix this JSONs and we will be good to go

File: extensions/google-translate/src/supportedLanguages.json [hunk 3]
```diff
@@ -70,8 +70,7 @@
   },
   "Mandarin": {
     "code": "zh-TW",
-    "name": "Taiwanese Mandarin",
-    "flag": "🇹🇼"
```
Comment (neighborhood999) [inline extensions/google-translate/src/supportedLanguages.json:74]: I don't know why the change needs to remove the flag.

Comment (y2-Whyto) [reply extensions/google-translate/src/supportedLanguages.json:74]: At the very first place, the flag of this language was written as the flag of PRC, and you changed it to the current flag afterwards. So if this is causing disputations, maybe it's a more neutral and acceptable way to just leave it blank I think. After all, the Traditional Chinese users are more diverse geographically. Leaving it blank doesn't make things worse.

File: extensions/google-translate/src/supportedLanguages.json (hunk 1 above)
Comment (y2-Whyto) [reply extensions/google-translate/src/supportedLanguages.json]: > Please fix this JSONs and we will be good to merge I think
> 
> (you can just approve my change suggestions)

Thank you. I didn't realize those tiny errors.

**Comment by y2-Whyto** [comment]
Hello @pernielsentikaer ,

Sorry for any possible disturbance.

Are you still maintaining this extension? I've made several modifications according to previous PR's. Also some brief discussions have been conducted.

So if you could kindly review my PR and give some suggestions (if any), I would be willingly cooperating.

Please let me know if there's anything I need to do.
